CustomDate - copia.js

```diff
@@ -0,0 +1,127 @@
+class CustomDate {
+    constructor() {
+        this.now = new Date();
+        this.date = new Date();
+        
+        //Use constants to avoid 'magic numbers'
+        this.daysOfTheWeek = ['SUNDAY', 'MONDAY', 'TUESDAY', 'WEDNESDAY', 'THURSDAY', 'FRIDAY', 'SATURDAY'];
```

Comment:
You don't need this, please read the documentation of you can do this `date.toLocaleDateString`